### PR TITLE
Fixes planted kudzu missing colour and desc

### PR DESCRIPTION
**Pull request by Razharas**
Fixes old kudzu bug
Fixes https://github.com/tgstation/tgstation/issues/9585

### CODE REVIEW

File: code/modules/events/spacevine.dm [hunk 1]
```diff
@@ -364,6 +368,18 @@
 	var/obj/effect/spacevine_controller/master = null
 	var/list/mutations = list()
 
+/obj/effect/spacevine/examine(mob/user)
+	..()
+	var/text = "This one is"
```
Comment (optimumtact) [inline code/modules/events/spacevine.dm]: "This one is normal vine."
    
    "This one is space protective fire vine."

I think you need to add an a

    "This one is a normal vine"
     
    "This one is a space protective fire vine"

File: code/modules/events/spacevine.dm [hunk 2]
```diff
@@ -491,19 +507,15 @@
 	vines += SV
 	SV.master = src
 	if(muts && muts.len)
-		SV.mutations |= muts
+		for(var/datum/spacevine_mutation/M in muts)
+			M.add(SV)
+		return
```
Comment (phil235) [inline code/modules/events/spacevine.dm]: are you sure about that return? won't this prevent new mutations to occur?

File: code/modules/events/spacevine.dm (hunk 1 above)
Comment (Razharas) [inline code/modules/events/spacevine.dm]: Me no english

File: code/modules/events/spacevine.dm (hunk 2 above)
Comment (Razharas) [inline code/modules/events/spacevine.dm]: If you specified the list of mutations new one needs to recieve you sure as hell dont want any random mutation to pop up

File: code/modules/events/spacevine.dm [hunk 3]
```diff
@@ -28,6 +28,10 @@
 	var/hue
 	var/quality
 
+/datum/spacevine_mutation/proc/add(obj/effect/spacevine/holder)
```
Comment (phil235) [inline code/modules/events/spacevine.dm]: not a fan of super generic proc names.

**Pull request merged**